prisma/schema.prisma

```diff
@@ -65,6 +65,13 @@ model GithubRepo {
     @@index([userId])
 }
 
+model Podcast {
+    id        String   @id @default(cuid())
+    source    String   @unique
+    user      User     @relation(fields: [userId], references: [id], onDelete: Cascade)
+    userId    String
+}
+
```

Comment:
For the podcast, I would recommend adding the following attributes:
* `url` instead of `source`: to be aligned with other models (should be a website or a podcast platform)
* `spotify`: URL of the podcast in Spotify (we are planning to use this to fetch the last episode from the Spotify RSS feed)
* `title`: the title of the podcast
* `description`: podcast description
* `draft`: by default to false for non admin users 
* `image`: podcast cover
* `createdAt` 
* `updatedAt` 


In the dashboard form, you only need to ask for `url` and `spotify`, but for other attributes, we need to extract them from Open Graph based on the `URL` attribute, just as we already did with blogs.

Check the blog table for more inspiration.

Reply:
Well received!